src/shims/shims.proj

```diff
@@ -18,7 +18,9 @@
         <StrongNameSig Condition="'%(NetfxReference.StrongNameSig)' == ''">StrongName</StrongNameSig>
         <StrongNameSig Condition="'%(NetfxReference.StrongNameSig)' != ''">%(NetfxReference.StrongNameSig)</StrongNameSig>
       </NetFxContracts>
-      <NETStandardContracts Include="$(NetStandardRefPath)netstandard.dll" />
+      <!-- TODO - once we retarget everything to netstandard2.1, remove netcoreapp2.0 reference to netstandard2.0 netstandard.dll -->
+      <NETStandardContracts Condition="'$(TargetGroup)' == 'netcoreapp2.0'" Include="$(NetStandardRefPath)netstandard.dll" />
```

Comment:
Should you rename `NetStandardRefPath` to `NetStandard20RefPath` everywhere? Otherwise it will be confusing forever. Anything generic to all versions can stay the same eg presumably `NETStandardContracts`

Reply:
Yes I think that's a good idea.

Reply:
Actually now I'm thinking it should stay as `NetStandardRefPath` - it points to `artifacts\bin\ref\netstandard`, and in the same location there is a `artifacts\bin\ref\netstandard2.0`. I think there'll be confusion either way, but the current state seems to more accurately describe what's going on.